mbs-s0014-20230920/README.md

```diff
@@ -0,0 +1,98 @@
+# Map-based noise simulations based on the SAT `mbs-noise-sims-sat` time-domain simulations
+
+Tag: `mbs-s0014-20230920`
+
+## Updates
+
+* 2023-09-20: first release.
+
+## Summary
+
+300 realistic realizations of the SAT map noise, estimated from the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain noise simulations. Simulations cover the full SAT footprint and utilize the tiled noise model.
+
+## Noise model
+
+This release is based on the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain simulations, available on `NERSC` at:
```

Comment:
it looks like this is a link to the software used for the sims, is there any link (even in the wiki not publicly available) to the documentation of the simulations?

Reply:
I can't find any wiki page about these TOD simulations. They were meant as an intermediate product to test the MBS pipeline. I added a list of specifications about the TOD sims to this page to make it clear what went in to the simulations.